backend/chainlit/message.py

```diff
@@ -206,6 +206,37 @@ class Message(MessageBase):
         elements (List[ElementBased], optional): A list of elements to send with the message.
     """
 
+    @classmethod
+    async def init(
+        cls,
+        content: Union[str, Dict],
+        author: Optional[str] = None,
+        language: Optional[str] = None,
+        actions: Optional[List[Action]] = None,
+        elements: Optional[List[ElementBased]] = None,
+        type: MessageStepType = "assistant_message",
+        metadata: Optional[Dict] = None,
+        tags: Optional[List[str]] = None,
+        id: Optional[str] = None,
+        parent_id: Optional[str] = None,
+        created_at: Union[str, None] = None,
+    ):
+        await asyncio.sleep(0.001)
+        self = cls(
+            content,
+            author,
+            language,
+            actions,
+            elements,
+            type,
+            metadata,
+            tags,
+            id,
+            parent_id,
+            created_at,
+        )
+        return self
+
```

Comment:
I don't see where this `init()` is called, here or anywhere. Could you please explain why this is here?

Reply:
Also note the type error raised by mypy.

Reply:
@dokterbob , as mentioned, if sleep is needed then we will need to initiate all Message or Step with `init`, which will be lots of changes. May I confirm on your side that sleep is absolutely needed before making a big change?

Reply:
I don't know why the sleep statement is here. Perhaps @willydouhard does?

I am asking about init() because you add this code to this PR and it does not seem to be called anywhere. I'd like to avoid dead code as much as possible.